app/authentication/flush_permission_denied.py

```diff
@@ -0,0 +1,8 @@
+
+class FlushPermissionDenied(Exception):
```

Comment:
This exception could be named something more generic, such as PermissionDeniedException, in order to prevent further exceptions from being created and attached to the same view.